images/prow-tests/Dockerfile

```diff
@@ -181,6 +181,7 @@ ARG GO_LICENSES_VERSION=v1.2.1
 ARG DOCKER_CREDENTIAL_GCR_VERSION=v2.0.5
 ARG JSONNET_VERSION=v0.18.0
 ARG COSIGN_VERSION=v1.13.0
+ARG TOOLBOX_VERSION=2aaf6de75f2e02cc078de98d4646d0803c8c5132
```

Comment:
we should just be using latest

Reply:
DO we still need this now that we have chris' change that auto adds latest toolbox and rebuilds with each change?